src/components/TheHeader.vue

```diff
@@ -12,17 +12,25 @@ export default defineComponent({
     HeaderNavigation,
     HeaderActions,
   },
+  computed: {
+    isNotAuthPanel() {
+      return !['LoginPage', 'RegisterPage'].includes(this.$route.name);
+    },
+  },
 });
 </script>
 
 <template>
   <header class="header">
-    <a href="#">
+    <router-link :to="{ name: 'HomePage' }">
       <LogoImage />
-    </a>
-    <HeaderNavigation class="header__navigation" />
-    <HeaderActions class="header__actions" />
-    <HamburgerButton class="header__hamburger" />
+    </router-link>
+
+    <HeaderNavigation v-if="isNotAuthPanel" class="header__navigation" />
+    <HeaderActions v-if="isNotAuthPanel" class="header__actions" />
+    <HamburgerButton v-if="isNotAuthPanel" class="header__hamburger" />
```

Comment:
You could have just one `v-if` and group everything under `template`.
The `template` element will not be displayed but can be used like this as an "invisible" wrapper
https://vuejs.org/guide/essentials/conditional.html#v-if-on-template
```suggestion
    <template v-if="isNotAuthPanel">
      <HeaderNavigation class="header__navigation" />
      <HeaderActions class="header__actions" />
      <HamburgerButton class="header__hamburger" />
    </template>
```

Reply:
Thank you so much for that! I really didn't want to use a wrapper because it would break my styles, but this is a brilliant way around that 😄